README.md

```diff
@@ -1,135 +1,104 @@
+
+1. [Sign up and log in](https://docs.camunda.io/docs/next/guides/getting-started/) at [https://camunda.io](https://camunda.io).
+2. [Create a new cluster](https://docs.camunda.io/docs/guides/create-cluster/) with the latest stable version of Zeebe and the name **Test cluster**.
 
 ![Camunda Platform Cluster](images/cluster.png)
 
-After the cluster is available, we need to create a pair of [Client
-Credentials](https://docs.camunda.io/docs/components/console/manage-clusters/manage-api-clients/).
-To do this, take the following steps:
+3. After the cluster is available, [create client credentials](https://docs.camunda.io/docs/components/console/manage-clusters/manage-api-clients/). To do this:
+   1. From Console, navigate to the **Clusters** detail page.
+   2. Click on your cluster and switch to the **API** tab.
+   3. Click **Create new client** and name your client accordingly.
 
-1. Navigate to the **Clusters** detail page.
-2. Switch to the **API** tab.
-3. Create a new client credential and either note down the **client id** and **client secret**, or
-download the credentials file.
+**NOTE: Ensure you keep the generated client credentials in a safe place. The client secret will not be shown again. For your convenience, you can also download the client information to your computer.**
 
 # Model the process
 
-In this example, we'll model a simple process to send an email message.
+In this example, we'll model a process to send an email message.
 
-Camunda Platform 8 allows you to use the
+Camunda 8 allows you to use the
 [BPMN](https://docs.camunda.io/docs/components/modeler/bpmn/bpmn-primer/)
 standard to model your business processes. The process consists of two
-tasks: a user task to allow a human to enter the message content, and a service
-task to automatically send the email message.
+tasks: a [user task](https://docs.camunda.io/docs/next/guides/orchestrate-human-tasks/) to allow a human to enter the message content, and a [service
+task](https://docs.camunda.io/docs/next/guides/orchestrate-microservices/) to automatically send the email message.
 
 ![Process](images/send-email.png)
 
-**Note**: To keep the scope of this guide easy to follow, we only build the
-scaffolding for this process automation example and will not actually send
-an email. This can be a task for you to extend the example and connect to a mail
-provider.
-
-To create the process, we use the desktop [Camunda Modeler] or [Web Modeler].
+**NOTE: To keep the scope of this guide limited, this guide only steps through the framework for this process automation example and will not actually send an email. If you would like to send an email, you may extend the example and [connect to a mail provider](https://docs.camunda.io/docs/next/guides/configuring-out-of-the-box-connectors/).**
 
-Find the finished process at [process/send-email.bpmn](process/send-email.bpmn).
+To create the process, use [Modeler](https://docs.camunda.io/docs/components/modeler/about-modeler/).
```

Comment:
This is pretty blunt. Do we want to walk users through this portion of the guide? I've added "Find the finished process at [process/send-email.bpmn](process/send-email.bpmn), and learn more about [designing a process using BPMN](https://docs.camunda.io/docs/next/guides/automating-a-process-using-bpmn/)." but this may not be enough.